### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by StudioZut** [gwu-libraries/omeka-home#14]
Check for accessibility.

**Comment by StudioZut** [comment]
Passes. Added missing alt tags. Font color contrast could be improved still.

**Comment by StudioZut** [comment]
Added aria-labels and changed text to "read more" on the under-descriptive "more" links. Passes with warnings: unclear links from the basic-slider tool nav, hidden elements (mostly from responsive design, false positive) and contrast check for the two "orange" text colors.
